### ISSUE TITLE
Can't compile for x64

### ISSUE THREAD

**Issue by KraXarN** [nexe/nexe#228]
Trying to compile a x64 app for Windows.
Tried passing "x64" in nodeVCBuildArgs, but is ignored and compiled as x86 anyway

**Comment by jaredallard** [comment]
Thanks for the report. Will look into it. On a sidenote, I'm 100% sure I've seen you somewhere...

**Comment by yasserf** [comment]
I got the same issue just now as well:

```
D:\git\deepstream.io\nexe\src\node\4.0.0\node-v4.0.0\node.sln.metaproj : error MSB4126: The specified solution configuration "Release|X86" is invalid. Please specify a valid solution configuration using the Configuration and Platform properties (e.g. MSBuild.exe Solution.sln /p:Configuration=Debug /p:Platform="Any CPU") or leave those properties blank to use the default solution configuration. [D:\git\deepstream.io\nexe\src\node\4.0.0\node-v4.0.0\node.sln]
....> ERROR: The release executable has not been generated. This indicates a failure in the build process. There is likely additional information above.
```

However the configuration seems to contain the correct variable:

```
{ 'target_defaults': { 'cflags': [],
                       'default_configuration': 'Release',
                       'defines': [],
                       'include_dirs': [],
                       'libraries': []},
  'variables': { 'asan': 0,
                 'host_arch': 'x64',
                 'icu_small': 'false',
                 'node_byteorder': 'little',
                 'node_install_npm': 'true',
                 'node_prefix': '/usr/local',
                 'node_release_urlbase': '',
                 'node_shared_http_parser': 'false',
                 'node_shared_libuv': 'false',
                 'node_shared_openssl': 'false',
                 'node_shared_zlib': 'false',
                 'node_tag': '',
                 'node_use_dtrace': 'false',
                 'node_use_etw': 'true',
                 'node_use_lttng': 'false',
                 'node_use_openssl': 'true',
                 'node_use_perfctr': 'true',
                 'openssl_fips': '',
                 'openssl_no_asm': 0,
                 'python': 'C:\\Python27\\python.exe',
                 'target_arch': 'ia32',
                 'v8_enable_gdbjit': 0,
                 'v8_enable_i18n_support': 0,
                 'v8_no_strict_aliasing': 1,
                 'v8_optimized_debug': 0,
                 'v8_random_seed': 0,
                 'v8_use_snapshot': 1,
                 'want_separate_host_toolset': 1}}
```

**Comment by KraXarN** [comment]
Realized this is pretty much a dupe of #195, but it's documented that you should be able to, when it's card coded. Also as in that issue, editing the exe.js directly makes it compile for x64

**Comment by yasserf** [comment]
The solution from #195 unfortunately did not work for me, still giving the same error i pasted above =(

**Comment by KraXarN** [comment]
Try upgrading your Node to 6.2.0 and if you want to compile 64-bit, only pass "x64". Release is already passed by default and x86 is 32-bit

**Comment by yasserf** [comment]
Used version `4.4.5` and that seemed to solve it! ( We want to use the lts version for our bundled issues ).

Now I just have to (continue) figuring out how to get appveyor to compile native modules.

Thanks for your help!

**Comment by alexandre1985** [comment]
If it is resolved you may consider closing this issue

**Comment by rahul-dutt-sharma** [comment]
If I am correct the buildargs were never getting used in the code from npm. I had to use the code from github to make it work.

**Comment by KraXarN** [comment]
Correct, not sure if that's still the case.

**Comment by calebboyd** [comment]
fixed in #308
